### PR TITLE
add oled esp pico d4, ADC changued

### PR DESCRIPTION
**Pull request by roberbike**
ADC oled change to curv = ((float)v * 1.1) / (4095);
Add esp32 pico d4 power sensor pin to 19, adc to 36.

### CODE REVIEW

File: lib/batterylib/battery_oled.cpp [hunk 1]
```diff
@@ -47,7 +47,8 @@ void Battery_OLED::update() {
     digitalWrite(ADC_EN, HIGH);
     delay(10);  // suggested by @ygator user in issue #2
     uint16_t v = analogRead(ADC_PIN);
-    curv = ((float)v / 4095.0) * 16;
+    // curv = ((float)v / 4095.0) * 16;
```
Comment (hpsaturn) [inline lib/batterylib/battery_oled.cpp:50]: please remove unnecessary comments

File: lib/batterylib/battery_oled.cpp [hunk 2]
```diff
@@ -47,7 +47,8 @@ void Battery_OLED::update() {
...
+    curv = ((float)v * 1.1) / (4095);
     digitalWrite(ADC_EN, LOW);  // for possible issue: https://github.com/Xinyuan-LilyGO/TTGO-T-Display/issues/6
     // return curv;
```
Comment (hpsaturn) [inline lib/batterylib/battery_oled.cpp:53]: please remove unnecessary comments

**Review by hpsaturn — CHANGES_REQUESTED** [review]

File: lib/batterylib/battery_oled.cpp (hunk 1 above)
Comment (roberbike) [reply lib/batterylib/battery_oled.cpp]: Done

File: lib/batterylib/battery_oled.cpp (hunk 2 above)
Comment (roberbike) [reply lib/batterylib/battery_oled.cpp]: Done

**Pull request merged**